async_upnp_client/client_factory.py

```diff
@@ -53,7 +53,7 @@ def __init__(
         requester: UpnpRequester,
         disable_state_variable_validation: bool = False,
         disable_unknown_out_argument_error: bool = False,
-        non_strict: bool = False,
+        non_strict: bool = True,
```

Comment:
Lets keep it `False` by default. Home assistant (or at least the dlna_dmr component) sets it to true already: https://github.com/home-assistant/core/blob/3d212b868ef746181d85a93ee430f4fc23f309d6/homeassistant/components/dlna_dmr/data.py#L41

Reply:
Yes, I saw, it was my mistake when editing file before commit.
The real (and interesting) commit is the one below from line 146.
Could you pls help me to delete line 56 from the diff? I'm really new to github ;(

Reply:
You can simply change it and commit+push it. Don't worry about changing it twice, the commits will get squashed to a single commit.